ch.threema.threema-web-desktop.yml

```diff
@@ -0,0 +1,74 @@
+app-id: ch.threema.threema-web-desktop
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-download
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+    DEV_ENV: development
```

Comment:
should that be something else since it's not a development build?

Reply:
Upstream confirmed that this is required "for now", but I wanted to document this oddity but forgot about it.

Reply:
Addressed this with a comment.